src/main/kotlin/test/ktortemplate/core/exception/AppException.kt

```diff
@@ -0,0 +1,9 @@
+package test.ktortemplate.core.exception
+
+class AppException(
+    val code: ErrorCode,
+    val id: Any? = null,
```

Comment:
Also I would avoid an extensive use of the Any type (it's basically an Object), as it tends to:
a) lead to issues with common tasks like serialization
b) usually signifies we're trying to foresee stuff. If we don't know what these fields are for, them remove them.

Reply:
payload can be set to string and assume the caller serialization. For id's I'm not so sure, id's can be int, int8, guid, string... So unless we have a way to ensure the data type somehow, I think Any it's the only option (or String and also pass the serialization to the caller)